### ISSUE TITLE
Listing and loading hub modules

### ISSUE THREAD

**Issue by exploide** [AdrianVollmer/PowerHub#40]
Currently, `payload.ps1` contains full paths to the available modules.
E.g. `/home/user/.local/share/powerhub/modules/ps1/PowerSploit/Recon/PowerView.ps1`.

Is this information really needed on the target side?
It contains details like the attacker's name and directories, which might be sensitive.

Furthermore the length of the full path can be a problem for the layout of the `lshm` table.
I have seen output where it got truncated at the end, cutting off the actual name of the script (making them indistinguishable).

I would like to propose to strip the module path from the module names.
E.g. the line from above becomes `ps1/PowerSploit/Recon/PowerView.ps1`.
That's both, shorter and more privacy preserving.

---

While talking about the `lshm` table:
From a user's perspective, I don't really know what the `code` column is about.
I only notice it has a value once the module is loaded.
Again, this column is so wide, it destroys the table layout, since it leads to a line wrap.

Since I guess the code is only an internal identifier, it would be great to not display the code.
Instead, a short column stating something like `loaded` would be great, to see which modules are already present.

---

(Then, I'm still confused why `lhm 8` does not work, while `lhm 08` does.
But since you already documented that in the help, I guess there is no easy fix.
So feel free to ignore this complain :P )

Thanks again for your work!

**Comment by AdrianVollmer** [comment]
All good points

>I'm still confused why lhm 8 does not work, while lhm 08 does

You and I both ;)

I never use the number anymore anyway. `lhm powerview` or whatever is way more convenient.

**Comment by AdrianVollmer** [comment]
Ah, it's because somewhere a string is compared with an integer:

`if ($i -lt $Modules.length ...`

With a leading zero the condition becomes accidentally correct. Fixed in 3f46ad36add5a886d9e2285d272d7f48b233c3c0

**Comment by exploide** [comment]
I think there is a regression here. I noticed the full path to the modules on the attacker's machine are back.

```
PS> lshm

N Type   Name                                                                Loade
- ----   ----                                                                -----
0 ps1    /home/user/.local/share/powerhub/modules/ps1/PrivescCheck.ps1       False
1 ps1    /home/user/.local/share/powerhub/modules/ps1/PrintNightmare-LPE.ps1 False
2 dotnet /home/user/.local/share/powerhub/modules/exe/SharpHound.exe         False
```
